app/main/views/frameworks.py

```diff
@@ -260,6 +260,17 @@ def framework_submission_services(framework_slug, lot_slug):
     if framework['status'] == 'pending' and declaration_status != 'complete':
         abort(404)
 
+    previous_services = data_api_client.find_services(
+        supplier_id=current_user.supplier_id,
+        framework='g-cloud-9',
```

Comment:
Should be `content_loader.get_metadata('g-cloud-10', 'copying-services', 'framework')` now - although will be best to hit the API to get these framework details for later use.